jena-base/src/main/java/org/apache/jena/atlas/lib/CollectionUtils.java

```diff
@@ -22,12 +22,7 @@
 
 public class CollectionUtils
 {   
-    public static <T> boolean disjoint(Collection<T> c1, Collection<T> c2)
-    {
-		return c1.stream().noneMatch(c2::contains);
-    }
-    
-    /** Test for same elements, regarless of cardinality */ 
+    /** Test for same elements, regardless of cardinality */ 
     public static <T> boolean sameElts(Collection<T> left, Collection<T> right) {
```

Comment:
You know, you would think there would be, but I didn't find anything that  ignores both cardinality and order and works for all `Collection`s. OTOH, I just check and nothing in Jena actually _uses_ this function except its own tests! So I wouldn't be averse to just dropping it. Anyone who was using it should have no difficulty in-lining it or inserting their own.